tools/matRad_spotRemoval.m

```diff
@@ -0,0 +1,99 @@
+function [dij,stf] = matRad_spotRemoval(dij,w,stf,thres)
```

Comment:
Can you call this spotRemovalDij?
I think this implementation would also be interesting in a class format. As there are multiple other thinkable ways of removing spots, and also multipble considerable output formats (logical index in w, for example, dij, direct postprocessing to resultGUI, etc.). But we might postpone this to another time.

Reply:
I added a first spotRemoval class which is able to use different modes to remove spots and then recalculate the resultGUI.